cmake/AddPdfTarget.cmake

```diff
@@ -86,10 +87,14 @@ function (add_pdf_target docname lang entities figures dtd_files)
         DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-xslticon-trigger"
                 "${gnucash_icon_files}")
 
+    add_custom_target("${lang}-${docname}-${fmt}"
+        DEPENDS "${OUTPUT_DIR}/${outfile}")
 
-    add_custom_target("${lang}-${docname}-pdf"
-        DEPENDS "${BUILD_DIR}/${pdffile}")
+    add_dependencies(${docname}-${fmt} "${lang}-${docname}-${fmt}")
 
-    add_dependencies(${docname}-pdf "${lang}-${docname}-pdf")
+    install(FILES "${OUTPUT_DIR}/${outfile}"
+        DESTINATION "share/doc/${lang}"
+        OPTIONAL
+        COMPONENT "${fmt}")
```

Comment:
As explained on an earlier PR we don't need install targets for pdf, epub and mobi. If you really want to add those, that should  go in a separate commit.

I hope by now you get the picture: one topic per commit. It's incredibly confusing trying to figure out why a certain change was made if a single commit mixes different goals.